warehouse/query-core/src/main/java/datawave/query/language/parser/jexl/LuceneToJexlQueryParser.java

```diff
@@ -86,6 +81,20 @@ public JexlNode convertToJexlNode(String query) throws ParseException {
         return parsedQuery;
     }
 
+    @Override
+    public org.apache.lucene.queryparser.flexible.core.nodes.QueryNode parseToLuceneQueryNode(String query) throws QueryNodeParseException {
+        query = query.replaceAll("\\u0093", "\""); // replace open smart quote 147
```

Comment:
Consider adding these to Constants as well?

Reply:
No problem, will do.